app/gui2/src/bindings.ts

```diff
@@ -38,6 +39,12 @@ export const graphBindings = defineKeybinds('graph-editor', {
   exitNode: ['Mod+Shift+E'],
 })
 
+export const visualizationBindings = defineKeybinds('visualization', {
+  nextType: ['Mod+Space'],
+  toggleFullscreen: ['Shift+Space'],
+  exitFullscreen: ['Escape'],
+})
+
```

Comment:
Instead of duplicating bindings, I would make GraphEditor implement all handlers for visualizationBindings in similar way as toggleFullscreen is implemented (check for single selected node).